modules/persistent-storage-csi-migration-enable.adoc

```diff
@@ -0,0 +1,59 @@
+// Module included in the following assemblies:
+//
+// * storage/container_storage_interface/persistent-storage-csi-migration.adoc
+
+[id="persistent-storage-csi-migration-enable_{context}"]
+= Enabling CSI automatic migration
+
+[NOTE]
+====
+Enabling CSI automatic migration drains, and then restarts, all nodes in the cluster in sequence. This might take some time.
+====
+
+.Procedure
+
+* Enable feature gates (see xref:../../nodes/clusters/nodes-cluster-enabling-features.adoc[Enabling features using feature gates]).
```

Comment:
* You can't use an xref from within a module. I'd instead add an "== Additional resources" section to the end of the assembly and link to it there.
* I don't think that "Enable feature gates" is the right terminology. Should it be something like "Enable CSI automatic migration by using a feature gate"?